contrib/devtools/dockerfile

```
@@ -1,6 +1,6 @@
 FROM bufbuild/buf:1.1.0 as BUILDER
 
-FROM golang:1.17-alpine
+FROM golang:1.18-alpine
```

```suggestion
FROM golang:1.17-alpine
```

need to keep this 1.17 for a little longer, go install can't work with cosmos-proto